### PR TITLE
tests: Improve messages related to ROX-22594.

### PR DESCRIPTION
**Pull request by porridge**
## Description

These are changes I was making when getting familiar with the test while investigating the flakes in ROX-22594. The flakes turned out to have an unrelated cause, but I think these improvements are worth merging anyway.

- improve messages and comments
- use consistent gerund form
- add a couple of log messages
- add a check for multiple clusters
- add comments to messages which are checked by tests

## Checklist
- [x] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~
- [x] ~Evaluated and added CHANGELOG entry if required~
- [x] ~Determined and documented upgrade steps~
- [x] ~Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

### Here I tell how I validated my change

CI is enough.

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: qa-tests-backend/src/test/groovy/TLSChallengeTest.groovy [hunk 1]
```diff
@@ -155,15 +156,16 @@ class TLSChallengeTest extends BaseSpecification {
         while (t.IsValid()) {
             List<ClusterOuterClass.Cluster> list = Services.getClusterClient().getClusters().getClustersList()
             if (list.empty) {
-                throw new OrchestratorManagerException(
-                        "Did not found any cluster, maybe redeploy StackRox or register a new cluster.")
+                throw new OrchestratorManagerException("Central dos not know about any secured clusters.")
```
Comment (vikin91) [inline qa-tests-backend/src/test/groovy/TLSChallengeTest.groovy:159]:
```suggestion
                throw new OrchestratorManagerException("Central does not know about any secured clusters.")
```

**Review by vikin91 — APPROVED** [review]
Good stuff! Caught one typo

**Review by gavin-stackrox — APPROVED** [review]

**Pull request merged**